.template/addons/crud/lib/template.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+def copy_template_files
+  ignore_tt do
+    directory 'lib/templates', renderTemplates: false
+  end
+end
+
+def ignore_tt
+  # NOTE: change template extension so it would skip
+  #       `when /#{TEMPLATE_EXTNAME}$/` condition and
+  #        fallback to default `copy_file`
+  Thor::TEMPLATE_EXTNAME.concat '_no_match' # => .tt_no_match
+  yield
+ensure
+  # NOTE: make sure to undo our dirty work after the block
+  Thor::TEMPLATE_EXTNAME.chomp! '_no_match' # => .tt
+end
```

Comment:
Should we move this method in a `ThorUtils` module in `.template/lib/thor_utils.rb`? 🤔

Reply:
Good idea. Thank you for the suggestion 🙏 

Updated in https://github.com/nimblehq/rails-templates/pull/505/commits/8efd9539c3643a723b13e13c088f990e9ffa5152